resources/prometheus/prometheus-prometheus.yaml

```diff
@@ -33,14 +33,18 @@ spec:
   retention: 30d
   affinity:
     podAntiAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-      - weight: 100
-        podAffinityTerm:
-          topologyKey: kubernetes.io/hostname
-          labelSelector:
-            matchLabels:
-              app: prometheus
-              prometheus: k8s
+      requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Hmm, I wonder if this stricter affinity rule will break single-node clusters somehow. E.g. presumably the second pod will always be "pending" with this change? Who knows what this will affect, possibly upgrades or other things that require all pods to be running. Same for similar alertmanager changes above.